packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -13,6 +13,21 @@ const cx = classNames.bind(styles);
 
 const Icon = <IconCalendar />;
 
+const dateFormats = [
+  'س‌س‌س‌س/ي‌ي/ش‌ش',
+  'TT.MM.JJJJ',
+  'DD/MM/YYYY',
+  'DD/MM/AAAA',
+  'DD.MM.YYYY',
+  'JJ/MM/AAAA',
+  'DD/MM/JJJJ',
+  'DD/MM/ÅÅÅÅ',
```

Comment:
Can we just get these date formats from the [translations files](https://github.com/cerner/terra-framework/blob/master/packages/terra-date-picker/translations/sv.json#L2)?

Reply:
Now that I look over it, this became unnecessary after some other changes, so this can be removed.

Reply:
Removed! [847d4f0](https://github.com/cerner/terra-framework/pull/1004/commits/847d4f05fecc8074996a2d504e3cc216973b6eef)